web-ng/tsconfig.app.json

```diff
@@ -9,10 +9,12 @@
     "src/polyfills.ts"
   ],
   "include": [
-    "src/**/*.ts"
+    "src/**/*.ts",
+    "src/**/**/*.ts"
```

Comment:
`**/` is already recursive.. are you sure this is needed?

Reply:
so i did it because my ng test was complaining that is does not see color-picker test file after merge but it does not seem to be a problem now

Reply:
could you approve again?:P